operator/pom.xml

```diff
@@ -183,17 +183,6 @@
               <sourceDirectory>src/main/resources/scripts</sourceDirectory>
             </configuration>
           </execution>
-          <execution>
```

Comment:
What is the replacement for this test execution? Is this test still being run?

Reply:
Another oops. Somehow I wound up with an integration error. I thought I had caught and fixed them all. Clearly not.